Cargo.toml

```diff
@@ -210,7 +210,7 @@ criterion-stats = "0.3.0"
 crossbeam-channel = "0.5.13"
 csv = "1.3.0"
 ctrlc = "3.4.5"
-curve25519-dalek = "3.2.1"
+curve25519-dalek = { git = "https://github.com/anza-xyz/curve25519-dalek.git", rev = "b500cdc2a920cd5bff9e2dd974d7b97349d61464" }
```

Comment:
See comments below. 
we directly set the dependency via git dependency. It is found the patch.crates-io
 #  is ineffective in other crate which wrongly picks the curve25519-dalek 3.2.1 which has constraint on the zeroize >= 1 < 1.4.